packages/grid/devspace.yaml

```diff
@@ -108,11 +122,10 @@ dev:
       - name: DEBUGGER_ENABLED
         value: "True"
     ports:
-      - port: "5678" # debugpy
+      - port: ${BACKEND_DEBUGPY_HOST_PORT}:5678 # debugpy
     sync:
       - path: ./backend/grid:/root/app/grid
       - path: ../syft:/root/app/syft
-    ssh: {}
```

Comment:
@yashgorana  I removed ssh, and it was creating port conflicts in multi cluster deployments.

Could you confirm, if it is not being used?

Reply:
It was used for debugging using ssh. We can tweak the port for this as well - https://www.devspace.sh/docs/configuration/dev/connections/ssh#config-reference

Reply:
Fixed by [8bc4b84](https://github.com/OpenMined/PySyft/pull/8610/commits/8bc4b8461c68a4cd74beb5f6d9c8dd47ad70433c)